src/client/components/voyage/InputField.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+
+class InputField extends Component {
+  render() {
+    const {
+      label,
+      name,
+      min,
+      max,
+      step,
+      unit,
+      value,
+      onDataInputChange
+    } = this.props;
+    return (
+      <div>
+        <label htmlFor="number-input-field">{label}</label>
+        <input
+          style={{ width: 100 }}
```

Comment:
We are not using style now but later maybe we will use Material UI

Reply:
Comment has been resolved.